src/styles/main.scss

```diff
@@ -1,3 +1,43 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  box-sizing: border-box;
+}
+
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
 }
+
+.container {
+  height: 100vh;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width: 1200px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+.block {
+  min-height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
This is task about using "grid". You should use **display: grid;** And then you can also use justify-content: center;
 align-items: center; because these properties are the same as in "flex"